internal/app/cloudwatch/usage.go

```diff
@@ -157,5 +166,7 @@ func (u *usageFetcher) GetUsageMetrics() (UsageMetrics, error) {
 		}
 	}
 
+	span.SetStatus(codes.Ok, "metrics fetched")
```

Comment:
The description will only be added in case of error here ([doc](https://github.com/open-telemetry/opentelemetry-go/blob/trace/v1.22.0/trace/trace.go#L380))

Maybe we want to add this as an event in the span?

Reply:
It won't be sent indeed, but the `description` parameter of `span.SetStatus()` is required.
So I used a helpful description of the span completion (instead of `span.SetStatus(codes.Ok, "")`).

My understanding is that [OpenTelemetry events](https://opentelemetry.io/docs/languages/go/instrumentation/#events) are used to reported "things that occured during a span". Here we want to mark span as sucess to avoid "unset" span status.

<img width="427" alt="screenshot 2024-02-01 at 11 35 43" src="https://github.com/qonto/prometheus-rds-exporter/assets/319005/b8799fa1-2c56-46dc-8b73-6ea5d7e0c28e">

Reply:
Perfect then! 

Yes, I saw this being sent as an event in other services, when you still have things to do after you fetched the data